revolv/static/js/base/accounting.js

```diff
@@ -0,0 +1,388 @@
+
+}
+
+/**
+ * Places a cash in AdminAdjustment element on the page.
+ * @param {Object} data_to_insert - A dictionary with values to insert.
+ * @param {String} time - The time period in which this was inserted.
+ */
+var allCashInTransactions = {};
+function placeCashInAdjustmentElement(data_to_insert, time) {
+	var total = data_to_insert['total'];
+	var transactions = data_to_insert['transactions'];
+
+	var allTransactionsKeys = Object.keys(allCashInTransactions);
+	for (var i = 0; i < allTransactionsKeys.length; i++) {
+		allCashInTransactions[allTransactionsKeys[i]] = 0;
+	}
+	//insert all the transactions
+	var transactionKeys = Object.keys(transactions)
+	for (var i = 0; i < transactionKeys.length; i++) {
+		var transactionName = transactionKeys[i];
+		var transactionAmount = transactions[transactionName];
+		var cleanedTransactionName = cleanUpString(transactionName);
+		allCashInTransactions[cleanedTransactionName] = transactionAmount;
+	}
+	
+	allTransactionsKeys = Object.keys(allCashInTransactions);
+	for (var i = 0; i < allTransactionsKeys.length; i++) {
+		var name = allTransactionsKeys[i];
+		var amount = allCashInTransactions[name];
+		$('.transaction').filter('.' + name).append('<td class="added">' + amount + "</td>");
+	}
+
+	$('.cash-in-adjustment-total').append('<td class="added">' + total + "</td>");
+}
+
+/**
+ * Places a cash out AdminAdjustment element on the page.
+ * @param {Object} data_to_insert - A dictionary with values to insert.
+ * @param {String} time - The time period in which this was inserted.
+ */
+var allCashOutTransactions = {};
+function placeCashOutAdjustmentElement(data_to_insert, time) {
+
+	var total = data_to_insert['total'];
+	var transactions = data_to_insert['transactions'];
+
+	var allTransactionsKeys = Object.keys(allCashOutTransactions);
+	for (var i = 0; i < allTransactionsKeys.length; i++) {
+		allCashOutTransactions[allTransactionsKeys[i]] = 0;
+	}
+	//insert all the transactions
+	var transactionKeys = Object.keys(transactions)
+	for (var i = 0; i < transactionKeys.length; i++) {
+		var transactionName = transactionKeys[i];
+		var transactionAmount = transactions[transactionName];
+		var cleanedTransactionName = cleanUpString(transactionName);
+		allCashOutTransactions[cleanedTransactionName] = transactionAmount;
+	}
+	
+	allTransactionsKeys = Object.keys(allCashOutTransactions);
+	for (var i = 0; i < allTransactionsKeys.length; i++) {
+		var name = allTransactionsKeys[i];
+		var amount = allCashOutTransactions[name];
+		$('.transaction').filter('.' + name).append('<td class="added">' + amount + "</td>");
+	}
+
+	$('.cash-out-adjustment-total').append('<td class="added">' + total + "</td>");
+}
+
+/**
+ * Iterates through the data and inserts values onto the page.
+ * @param {Object} data - A dictionary of all the data
+ * @param {String} cash_in_or_cash_out - 'cash_in' for cash in, and 'cash_out' for cash out.
```

Comment:
Do you think it would be better to just have a boolean where true -> cash in, false -> cash out?